### PR TITLE
Improve generated doc comments for goPathChildConstructorTemplate.

### LINKED ISSUE #591: [ypathgen]: goPathChildConstructorTemplate needs a better docstring for wildcards
Right now it generates the same comment which is confusing.

### PR DESCRIPTION
**Pull request by wenovus**
Added defining module (as opposed to instantiating module, which is seen in the "Path from root" abolute path), absolute path (no key), YANG node type, and YANG description.

Also remove testing of ChildConstructor doc comments in most tests in pathgen_test.go to reduce repetitive testing.

FIXED #591 

Example:
```go
 68479 // OperStatus (leaf): The current operational state of the interface.                                                                                                                                                                                                                                                                               
 68480 //                                                                                                                                                                                                                                                                                                                                                  
 68481 // This leaf has the same semantics as ifOperStatus.                                                                                                                                                                                                                                                                                                
 68482 // ----------------------------------------                                                                                                                                                                                                                                                                                                         
 68483 // Defining module: "openconfig-interfaces"                                                                                                                                                                                                                                                                                                         
 68484 // Instantiating module: "openconfig-interfaces"                                                                                                                                                                                                                                                                                                    
 68485 // Path from parent: "state/oper-status"                                                                                                                                                                                                                                                                                                            
 68486 // Path from root: "/interfaces/interface/state/oper-status"                                                                                                                                                                                                                                                                                        
 68487 func (n *InterfacePath) OperStatus() *Interface_OperStatusPath {                                                                                                                                                                                                                                                                                    
 68488         return &Interface_OperStatusPath{                                                                                                                                                                                                                                                                                                           
 68489                 NodePath: ygot.NewNodePath(                                                                                                                                                                                                                                                                                                         
 68490                         []string{"state", "oper-status"},                                                                                                                                                                                                                                                                                           
 68491                         map[string]interface{}{},                                                                                                                                                                                                                                                                                                   
 68492                         n,                                                                                                                                                                                                                                                                                                                          
 68493                 ),                                                                                                                                                                                                                                                                                                                                  
 68494         }                                                                                                                                                                                                                                                                                                                                           
 68495 }                                                                                                                                                                                                                                                                                                                                                   
```

### CODE REVIEW

**Comment by robshakir** [comment]
The path from root specified here is incorrect -- you need to trim element 0 from the slice as this is the module name.

File: ypathgen/pathgen.go [hunk 1]
```diff
@@ -940,13 +949,29 @@ func generateChildConstructors(methodBuf *strings.Builder, builderBuf *strings.B
...
-		Struct:           structData,
-		RelPathList:      `"` + strings.Join(relPath, `", "`) + `"`,
-		ChildPkgAccessor: childPkgAccessor,
+		MethodName:         goFieldName,
+		TypeName:           fieldTypeName,
+		SchemaName:         field.Name,
+		YANGNodeType:       YANGNodeType,
+		YANGDescription:    strings.ReplaceAll(field.Description, "\n", "\n// "),
+		DefiningModuleName: definingModuleName,
+		AbsPath:            field.Path(),
```
Comment (robshakir) [inline ypathgen/pathgen.go:970]: Per my other comment, here's where we need to do `field.Path()[1:]`

Comment (wenovus) [reply ypathgen/pathgen.go]: Used a function derived from `SchemaPathNoChoiceCase` to keep it consistent with `RelPath`.

Also added comment line for instantiating module.

**Review by robshakir — APPROVED** [review]
Thanks for the changes!

**Pull request merged**